components/swatch/index.css

```diff
@@ -40,12 +40,14 @@
 	/* Light theme placeholder tokens */
 	.spectrum--light & {
 		--spectrum-swatch-border-color: rgba(0, 0, 0, 51%);
+		--spectrum-swatch-border-color-light: rgba(0, 0, 0, 20%);
```

Comment:
Based on some previous Slack discussions about variable naming, I think it might be a good idea to swap the terms around and rename the variable to something like:
`--spectrum-swatch-light-border-color`